x-pack/plugins/lists/common/schemas/common/schemas.test.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { pipe } from 'fp-ts/lib/pipeable';
+import { left } from 'fp-ts/lib/Either';
+
+import { foldLeftRight, getPaths } from '../../siem_common_deps';
+
+import { operator_type as operatorType } from './schemas';
+
+describe('Common schemas', () => {
+  describe('operatorType', () => {
```

Comment:
@FrankHassanabad like we talked about, added these tests, so if the type is updated by anyone, they should have to update these tests and see the note about needing to update the Typescript enum as well.